libethereum/Client.cpp

```diff
@@ -653,7 +654,7 @@ void Client::resyncStateFromChain()
 
 void Client::resetState()
 {
-	State newPreMine;
+	Block newPreMine;
```

Comment:
similiar, not tested just a compile fix which I think is correct. Review carefully.